Documentation/gettingstarted/requirements-eks.rst

```diff
@@ -16,36 +16,6 @@ For more information on AWS ENI mode, see :ref:`ipam_eni`.
    If you want to chain Cilium on top of the AWS CNI, refer to the guide
    :ref:`chaining_aws_cni`.
 
-The following command creates a Kubernetes cluster with ``eksctl``
-using `Amazon Elastic Kubernetes Service
-<https://aws.amazon.com/eks/>`_.  See `eksctl Installation
-<https://github.com/weaveworks/eksctl>`_ for instructions on how to
-install ``eksctl`` and prepare your account.
-
-.. code-block:: shell-session
-
-   export NAME="$(whoami)-$RANDOM"
-   cat <<EOF >eks-config.yaml
-   apiVersion: eksctl.io/v1alpha5
-   kind: ClusterConfig
-
-   metadata:
-     name: ${NAME}
-     region: eu-west-1
-
-   managedNodeGroups:
-   - name: ng-1
-     desiredCapacity: 2
-     privateNetworking: true
-     # taint nodes so that application pods are
-     # not scheduled until Cilium is deployed.
-     taints:
-      - key: "node.cilium.io/agent-not-ready"
-        value: "true"
-        effect: "NoSchedule"
-   EOF
-   eksctl create cluster -f ./eks-config.yaml
-
```

Comment:
> […]  these steps are not present in Documentation/gettingstarted/k8s-install-default.rst

They are already present in the [Create the Cluster](https://docs.cilium.io/en/latest/gettingstarted/k8s-install-default/#create-the-cluster) section.

Reply:
_Puts on glasses_
OK, all good :)

Reply:
I think needs to be kept otherwise this GSG won't have instructions for EKS https://docs.cilium.io/en/v1.10/gettingstarted/k8s-install-helm/

Reply:
@aanm as the commit message state it's a feature of the PR since we currently have cluster creation instructions only for EKS in the "Installation using Helm" guide.

Reply:
I might be missing something but EKS is not the only one that has creation instructions, AKS and RKE as well although they are disguised as "Requirements"

Reply:
@aanm Fair point! I added a Requirements section to EKS, please take another look.